models/ticketingDao.js

```diff
@@ -0,0 +1,50 @@
+const appDataSource = require('./dataSource');
+
+const getMovies = async () => {
+  try {
+    const rawQuery = `
+    SELECT DISTINCT
+      m.id as id,
+      m.name as movieTitle,
+      m.thumbnail_image_url as movieThumbnail,
+      DATE_FORMAT(m.opening_date, '%Y-%m-%d') as releaseDate,
+      fr.rating as rating
+    FROM movies m
+    JOIN film_ratings fr ON m.film_rating_id = fr.id
+    ORDER BY m.name ASC;
+    `;
+    const result = await appDataSource.query(rawQuery);
+    return result;
+  } catch (err) {
+    throw err;
+  }
+};
+
+const getTimetables = async () => {
+  try {
+    const rawQuery = `
+    SELECT
+      m.id as movieId,
+      tt.id as timetableId,
+      DATE_FORMAT(tt.date, '%Y-%m-%d') as date,
+      tt.time as time,
+      SUM(CASE WHEN tss.name  = '예매 가능' THEN 1 ELSE 0 END) as availableSeats,
+      COUNT(ts.id) as totalSeats
+    FROM time_tables_theater_seats ttts
+    JOIN theater_seat_status tss ON tss.id = ttts.theater_seat_status_id
+    JOIN theater_seats ts ON ts.id = ttts.theater_seat_id
+    JOIN theaters t ON t.id = ts.theater_id
+    JOIN time_tables tt ON tt.id = ttts.time_table_id
+    JOIN movies m ON m.id = tt.movie_id
+    WHERE tss.id != 1
```

Comment:
```suggestion
    WHERE tss.id != 1
```
`tss.id!=1` 과 같이 하드코딩되는것은 좋지 않습니다. 소프트웨어의 유지 보수가 필요한 시점에 작성된 코드를 누가 보더라도 읽기 쉽고 파악하기 용이한 코드가 깔끔한 코드라고 볼 수 있습니다. 가급적이면 하드코딩보다는 다른 방법을 활용해보세요! 해당 코멘트의 수정이 어렵다면 언제든지 질문주세요-!